sdv/constraints/tabular.py

```diff
@@ -442,8 +442,11 @@ def is_valid(self, table_data):
         """
         low = self._get_value(table_data, 'low')
         high = self._get_value(table_data, 'high')
+        isnull = np.logical_or(np.isnan(low), np.isnan(high))
 
-        return self.operator(high, low).all(axis=1)
+        valid = np.logical_or(self.operator(high, low), isnull)
```

Comment:
Too bad there is no way for the operator to ignore nulls.

Reply:
Yeah, it would be nice if there was a built-in option to ignore nulls.